ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1388,7 +1397,7 @@ IE_GPU.onnx_dyn_shapes_model_acosh_1_3
 IE_GPU.onnx_dyn_shapes_model_acosh_3_2
 IE_GPU.onnx_dyn_shapes_model_asinh_1_3
 IE_GPU.onnx_dyn_shapes_model_asinh_3_2
-IE_GPU.onnx_upsample9_scales_input_nearest_infer
+# IE_GPU.onnx_upsample9_scales_input_nearest_infer
```

Comment:
If you enabled this test, you can remove it from manifest

Reply:
Created ticket to enable this test.